### DISCUSSION TITLE
Add GPT Researcher to a Make.com step inside an escenario

### DISCUSSION THREAD

**Discussion by elegir** [assafelovic/gpt-researcher#323] [Q&A]
Hello! Anyone was able to add GPT researcher into a [Make.com](https://www.make.com/en) step for using it inside a scenario?

**Comment by viktorkacenak** [comment]
Yes. You have to deploy your own GPT researcher somewhere (Render, Railway, Vercel). You have to modify the app a bit to return the research to a webhook link you specify in the API request. So in the scenario you use HTTPS Module to send your query to your app and then have another scenario with Webhook trigger that will receive the research result. If you use Airtable or something to store the research, when calling the research initially it returns the task-id which you can store and then you can easily match it when you receive the research

**Comment by elegir** [comment]
Hey hello there! Martin here!

I just noticed this message. So sorry for not answering. First, thanks for
taking time t writing me about this!

Secondly, do you have any current version working with this perhaps?

On Mon, Jun 9, 2025 at 6:47 AM viktorkacenak ***@***.***>
wrote:

> Yes. You have to deploy your own GPT researcher somewhere (Render,
> Railway, Vercel). You have to modify the app a bit to return the research
> to a webhook link you specify in the API request. So in the scenario you
> use HTTPS Module to send your query to your app and then have another
> scenario with Webhook trigger that will receive the research result. If you
> use Airtable or something to store the research, when calling the research
> initially it returns the task-id which you can store and then you can
> easily match it when you receive the research
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/assafelovic/gpt-researcher/discussions/323#discussioncomment-13409822>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKJQIUJTZX23A4PINOLOUN33CV67JAVCNFSM6AAAAAB643H3G6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGNBQHE4DEMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Saludos / Best Regards

Martin Weidemann
